backends/edxorg.py

```diff
@@ -15,11 +16,25 @@ class EdxOrgOAuth2(BaseOAuth2):
     ID_KEY = 'edx_id'
     REQUEST_TOKEN_URL = None
     EDXORG_BASE_URL = settings.EDXORG_BASE_URL
+
+    # Settings for Django OAUTH toolkit
+    # AUTHORIZATION_URL = urljoin(EDXORG_BASE_URL, '/_o/authorize/')
+    # ACCESS_TOKEN_URL = urljoin(EDXORG_BASE_URL, '/_o/token/')
+    # DEFAULT_SCOPE = ['read', 'write']
+
+    # Settings for old edx OAUTH library
     AUTHORIZATION_URL = urljoin(EDXORG_BASE_URL, '/oauth2/authorize/')
     ACCESS_TOKEN_URL = urljoin(EDXORG_BASE_URL, '/oauth2/access_token/')
+    DEFAULT_SCOPE = ['email', ]
+
     ACCESS_TOKEN_METHOD = 'POST'
     REDIRECT_STATE = False
-    DEFAULT_SCOPE = ['email']
+    EXTRA_DATA = [
```

Comment:
It is used by the `'social.pipeline.social_auth.load_extra_data'`function in the `python social auth` pipeline: this is just configuration, the code is from the library itself.